Source/diablo.cpp

```diff
@@ -110,6 +113,19 @@ int color_cycle_timer;
 uint16_t gnTickDelay = 50;
 /** Game options */
 Options sgOptions;
+Keymapper keymapper {
+	// Workaround: remove once the INI library has been replaced.
+	[](const std::string &key, const std::string &value) {
+	    setIniValue("Controller.Keymapping", key.c_str(), value.c_str());

```

Comment:
Does this also work for controller input, or should we maybe call it "Keyboard" instead?

Reply:
For now it only maps keyboard keys, but could be extended to support controllers. Not sure why I called it like that now though. Happy to rename it, but why not `Keymapping`?

Reply:
Let's do that.